components/console_simple_init/.cz.yaml

```diff
@@ -0,0 +1,8 @@
+---
+commitizen:
+  bump_message: 'bump(console_simple_init): $current_version -> $new_version'
+  pre_bump_hooks: python ../../ci/changelog.py console_simple_init
+  tag_format: console_simple_init-v$version
+  version: 1.0.0
```

Comment:
would it trigger the master jobs now? I know that it's a bit tricky, since version `1.0.0` is already on master, but it didn't succeed so we probably need to update it?